python/core/auto_generated/qgsrendercontext.sip.in

```diff
@@ -111,6 +111,11 @@ Returns the destination QPainter for the render operation.
 %Docstring
 Returns the current coordinate transform for the context, or an invalid
 transform is no coordinate transformation is required.
+
+This usually represents the coordinate transform required to transform a layer
```

Comment:
I find "usually" a tricky term in an API. I.e. can I trust this behavior and if not, under what conditions not / what do I need to do.